fhir-r-lang/exercise_1.Rmd

```diff
@@ -0,0 +1,463 @@
+---
+title: "FHIR for Research - Exercise 1: Patients Prescribed Opioids (R version)"
+output:
+  html_document:
+    df_print: paged
+---
+
+## Learning Objectives and Key Concepts
+
+Workshop attendees will learn how to:
+* Query a FHIR Server to learn about its capabilities
```

Comment:
```suggestion

* Query a FHIR Server to learn about its capabilities
```

Looks like you need an extra line break here to get the bullets to show correctly.

![Screen Shot 2022-01-31 at 11 10 38 AM](https://user-images.githubusercontent.com/41651655/151830050-7cd9596f-ed14-4a94-8795-18aa58c97ef0.png)